packages/grafana-data/src/utils/valueMappings.ts

```diff
@@ -93,3 +95,9 @@ const isNullValueMap = (mapping: ValueMap): boolean => {
   }
   return mapping.value.toLowerCase() === 'null';
 };
+
+// Ref https://stackoverflow.com/a/42356340
+export function isNumeric(num: any) {
+  num = '' + num;
+  return !isNaN(num as number) && !isNaN(parseFloat(num));
+}
```

Comment:
I guess this would work, at least all the tests are passing 😉 
```suggestion
export function isNumeric(num: any) {
  if (num === true) {
    return false;
  }

  return Boolean(Number(num));
}
```